packages/perseus-editor/src/item-extras-editor.tsx

```diff
@@ -20,159 +26,208 @@ class ItemExtrasEditor extends React.Component<Props> {
+                        checked={this.props.calculator}
+                        onChange={(e) => {
+                            this.props.onChange({
+                                calculator: e.target.checked,
+                            });
+                        }}
+                    />
 
-                    {this.props.periodicTable && (
-                        <div>
-                            <label>
-                                Include key/legend with periodic table:{" "}
-                                <input
-                                    type="checkbox"
-                                    checked={this.props.periodicTableWithKey}
-                                    onChange={(e) => {
-                                        this.props.onChange({
-                                            periodicTableWithKey:
-                                                e.target.checked,
-                                        });
-                                    }}
-                                />
-                            </label>
-                            <InfoTip>
-                                Include a key for HS courses, omit for AP
-                                chemistry.
-                            </InfoTip>
-                        </div>
-                    )}
+                    <ItemExtraCheckbox
+                        label="Show financial calculator:"
+                        infoTip="This provides the student with the ability to view a financial calculator, e.g., for answering financial questions. Once checked, requires at least one of the three options below to be checked."
+                        checked={this.props.financialCalculator}
+                        onChange={(e) => {
+                            this.props.onChange({
+                                financialCalculator: e.target.checked,
+                                // If the financial calculator is unchecked,
+                                // this needs to be reset. All checked by
+                                // default.
+                                financialCalculatorMonthlyPayment:
+                                    e.target.checked,
+                                financialCalculatorTotalAmount:
+                                    e.target.checked,
+                                financialCalculatorTimeToPayOff:
+                                    e.target.checked,
+                            });
+                        }}
+                    />
 
-                    <div>
-                        <label>
-                            Show z table (statistics):{" "}
-                            <input
-                                type="checkbox"
-                                checked={this.props.zTable}
+                    {this.props.financialCalculator && (
```

Comment:
**question**: Are these three sub-types mutually exclusive?  If so, would it make sense for them to be a set of radio buttons instead?

Reply:
They are not mutually exclusive!

Reply:
Ah, then this makes sense.  Thanks.